### DISCUSSION TITLE
How to read named fields?

### DISCUSSION THREAD

**Discussion by SchmidtClaudia** [PHPOffice/PhpSpreadsheet#2314] [Q&A] [answered]
Hi

I have an existing spreadsheet with named fields spread over different worksheets that was created with Microsoft Excel. Without knowing or predefining the names I want to read only those fields from the worksheets. In the docs I only found to set those names and read the cells again. Is it possible to get a list of the named fields and there mapping cells, like the name manager in Excel can do?

Best regards

**Comment by oleibman** [comment, marked as answer]
```php
$spreadsheet->getNamedRanges();
```
